src/mo_frontend/assertions.mly

```diff
@@ -19,5 +19,7 @@ when I write this here (instead of in parser.mly)
     { AssertE(Precondition, e) @? at $sloc }
   | ASSERT COLON RETURN e=exp_nest
     { AssertE(Postcondition, e) @? at $sloc }
+  | ASSERT COLON s=NAT COLON ASYNC e=exp_nest
```

Comment:
```suggestion
  | ASSERT COLON s=NAT COLON ASYNC e=exp_nest
```
why are we adding the nat again if we always require it to be 1 anyway?

Reply:
Good q. Would that involve defining a new token? Too much hassle for very transient syntax.

Reply:
I just mean simplify to:
` 
| ASSERT COLON ASYNC e=exp_nest
`

Reply:
Aaah, but we may figure out what to do for `assert:2:async` (etc.) on Monday...